python/tvm/relay/quantize/quantize.py

```diff
@@ -150,6 +151,9 @@ def qconfig(**kwargs):
         Whether add stop_fusion when casting to dtype_activation. stop_fusion forces lowbit
         results to be stored in memory.
 
+    quantize_dense: boolean
+        Whether to quantize dense layers.
```

Comment:
maybe a better way is like that in autotvm, i.e., `quantize_op = [relay.conv2d, relay.dense]`?

Reply:
@yzhliu updated